manage-data/lifecycle/index-lifecycle-management/tutorial-automate-rollover.md

```diff
@@ -11,9 +11,12 @@ products:
 
 When you continuously index timestamped documents into {{es}}, you typically use a [data stream](../../data-store/data-streams.md) so you can periodically [roll over](rollover.md) to a new index. This enables you to implement a [hot-warm-cold architecture](../data-tiers.md) to meet your performance requirements for your newest data, control costs over time, enforce retention policies, and still get the most out of your data.
 
-::::{tip}
-[Data streams](../../data-store/data-streams.md) are best suited for [append-only](../../data-store/data-streams.md#data-streams-append-only) use cases. If you need to update or delete existing time series data, you can perform update or delete operations directly on the data stream backing index. If you frequently send multiple documents using the same `_id` expecting last-write-wins, you may want to use an index alias with a write index instead. You can still use [ILM](/manage-data/lifecycle/index-lifecycle-management/tutorial-automate-rollover.md) to manage and [roll over](rollover.md) the alias’s indices. Skip to [Manage time series data without data streams](/manage-data/lifecycle/index-lifecycle-management/tutorial-automate-rollover.md#manage-time-series-data-without-data-streams).
-::::
+To simplify index management and [automate rollover](/manage-data/lifecycle/index-lifecycle-management/rollover.md#ilm-automatic-rollover), select one of the scenarios that best applies to your situation:
+
+* When ingesting write-once, timestamped data that doesn't change, follow the steps in [Manage time series data with data streams](/manage-data/lifecycle/index-lifecycle-management/tutorial-automate-rollover.md#manage-time-series-data-with-data-streams) for simple, automated data stream rollover. ILM-managed backing indices are automatically created under a single data stream alias. ILM also tracks and transitions the backing indices through the lifecycle automatically. 
+* [Data streams](../../data-store/data-streams.md) are best suited for [append-only](../../data-store/data-streams.md#data-streams-append-only) use cases. If you need to update or delete existing time series data, you can perform update or delete operations directly on the data stream backing index. If you frequently send multiple documents using the same `_id` expecting last-write-wins, you may want to use an index alias with a write index instead. You can still use [ILM](/manage-data/lifecycle/index-lifecycle-management/tutorial-automate-rollover.md) to manage and [roll over](rollover.md) the alias’s indices. Skip to [Manage time series data without data streams](/manage-data/lifecycle/index-lifecycle-management/tutorial-automate-rollover.md#manage-time-series-data-without-data-streams).
```

Comment:
This sentence is too complex for the introduction. The objective of this bullet is to ensure the user understand that if they want to manage REGULAR INDICES (not data streams) and apply rollover, they need to XXX.

My proposal (if we want to leave that original sentence is to keep that in a separate note / tip).

I'll ping you in private to check options, but the introduction should clearly identify the 3 tutorials in this way:

- Implement rollover for Data Streams
- Implement rollover for regular indices (not data streams).
- Implement rollover for data without timestamp.

That's what we need to transmit somehow, with nice wording but being clear and straight.

Reply:
Addressed in a separate commit, thank you.